lua/nvimesweeper/ui.lua

```diff
@@ -342,7 +342,7 @@ local function move_cursor_to_click()
 end
 
 function M.new_ui(game, open_tab)
-  local buf = api.nvim_create_buf(true, true)
+  local buf = api.nvim_create_buf(false, true)
```

Comment:
Thanks for the PR!

While I think this change is fine for games started in the floating window (as the buffer will be wiped once it's hidden anyway; `bufhidden=wipe`), I don't believe it makes sense to unlist the buffer for games started in a tabpage, as `bufhidden` isn't set for those (for example, the game will still be running in the background even if the tabpage is closed, so I think it's valuable to have some indicator that the buffer still exists rather than relying on the potentially noisy output of `:ls!`).

As such, I would suggest the following change:

```suggestion
  local buf = api.nvim_create_buf(open_tab, true)
```

Does this sound reasonable?